### PR TITLE
Stubs

### PR DESCRIPTION
**Pull request by francismarcus**
A few stubs in regards to: #298

### CODE REVIEW

File: exercises/clock/clock.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Clock {
+  constructor(/* Parameters go here */) {
+  }
+  // Your code goes here
```
Comment (Roshanjossey) [inline exercises/clock/clock.ts]: I think it makes sense to have methods `plus`, `minus`, `equals` in the stub

File: exercises/grade-school/grade-school.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class GradeSchool {
+    // Your code goes here
```
Comment (Roshanjossey) [inline exercises/grade-school/grade-school.ts]: I think it'd be good to add methods named `addStudent`, `studentRoster`, `studentInGrade`

**Comment by Roshanjossey** [comment]
Hi @francismarcus, looking at stubs in other language tracks, I think it makes sense to add skeleton for methods that gets called in the tests.

**Comment by SleeplessByte** [comment]
> I think it makes sense to add skeleton for methods that gets called in the tests.

I agree with this. _tested_ methods should be in the stub. You can leave out the commentary (parameters here).

Perhaps we also throw, like in most language tracks? 

```typescript
throw new Error('Implement this function')
```

**Comment by francismarcus** [comment]
>  I think it makes sense to add skeleton for methods that gets called in the tests.

Agreed, makes sense :)

**Comment by masters3d** [comment]
Lgtm

**Comment by SleeplessByte** [comment]
We have stubs now.

**Pull request closed without merging**